.github/workflows/test.yaml

```diff
@@ -39,6 +39,37 @@ jobs:
         env:
           SLACK_WEBHOOK_URL_WORKFLOWS: ${{ secrets.SLACK_WEBHOOK_URL_WORKFLOWS }}
 
+  test-failure:
+    runs-on: ubuntu-latest
+    env:
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+      SLACK_WEBHOOK_URL_FAILING: ${{ secrets.SLACK_WEBHOOK_URL_FAILING }}
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Install Dependencies
+        run: yarn install
+
+      - uses: actions/setup-node@v1
+        with:
+          version: 12
+
+      - name: test on ${{ matrix.os }}
+        id: run-test
+        uses: nick-invision/retry@v1
+        continue-on-error: true
+        with:
+          timeout_minutes: 10
+          max_attempts: 3
+          command: bash .github/scripts/test-project.sh test test
+
+      - name: outcome
+        run: echo ${{ steps.run-test.outcome }}
+
+      - name: notify-slack
+        if: ${{ steps.run-test.outcome != 'success' }}
+        run: sh .github/slack/notify-failure.sh test test test
```

Comment:
TODO: Remove this test scenario before the merge, it looks like this

![image](https://user-images.githubusercontent.com/746482/87658484-e2e74900-c779-11ea-9dbc-23bee35dbf4c.png)